src/packages/tauv_common/src/eliz_introtask/camera_listener.py

```diff
@@ -0,0 +1,66 @@
+import rospy
+
+from sensor_msgs.msg import CameraInfo
+from sensor_msgs.msg import Image
+from tauv_msgs.srv import GetImageMask, GetImageMaskResponse, GetImageMaskRequest
+from std_srvs.srv import Trigger, TriggerResponse,TriggerRequest
+import cv2 as cv
+from cv_bridge import CvBridge
+
+class redDetection:
+    def __init__(self):
+        ##ned to figure out how to pass the image message to threhold image
+        self._sub = rospy.Subscriber('/kf/vehicle/oakd_bottom/color/image_raw',Image,self._handle_image)
```

Comment:
Shouldn't need to hardcode this. Using `vehicle/oakd_bottom/...` (no leading `/`) will automatically prefix the `/kf/` using ROS namespacing if the node is launched from a launch file correctly